packages/language-server/src/server.ts

```diff
@@ -46,41 +41,32 @@ function getConnection(options?: LSOptions): IConnection {
   return connection
 }
 
+let hasCodeActionLiteralsCapability = false
+let hasConfigurationCapability = false
+let defaultBinPath = ''
+
 /**
  * Starts the language server.
  *
  * @param options Options to customize behavior
  */
-export function startServer(options?: LSOptions): void {
+export function startServer(options?: LSPOptions): void {
   const connection: IConnection = getConnection(options)
   const documents: TextDocuments<TextDocument> = new TextDocuments(TextDocument)
 
-  // Does the clients accepts diagnostics with related information?
-  let hasCodeActionLiteralsCapability = false
-
   connection.onInitialize(async (params: InitializeParams) => {
     initializeTelemetry(connection)
     const capabilities = params.capabilities
 
     hasCodeActionLiteralsCapability = Boolean(
       capabilities?.textDocument?.codeAction?.codeActionLiteralSupport,
     )
+    hasConfigurationCapability = Boolean(capabilities?.workspace?.configuration)
 
-    const binPathPrismaFmt = await util.getBinPath()
-    if (await util.binaryIsNeeded(binPathPrismaFmt)) {
-      try {
-        await install(binPathPrismaFmt)
-        connection.console.info(
-          `Prisma plugin prisma-fmt installation succeeded.`,
-        )
-      } catch (err) {
-        sendException(await getSignature(), err, `Cannot install prisma-fmt.`)
-        connection.console.error('Cannot install prisma-fmt: ' + err) // eslint-disable-line @typescript-eslint/restrict-plus-operands
-      }
-    }
+    defaultBinPath = await util.getBinPath()
 
     connection.console.info(
-      `Installed version of Prisma binary 'prisma-fmt': ${util.getVersion()}`,
+      `Default version of Prisma binary 'prisma-fmt': ${util.getVersion()}`,
```

Comment:
Do we have a way to log the version of the `prisma-fmt` binary supplied by the path? If no, I would love to create a feature request for such an extension command.

Reply:
Yes, there is a way to do that already, I'll integrate this here.